wren-ui/src/components/pages/setup/dataSources/MySQLProperties.tsx

```
@@ -1,15 +1,73 @@
+      }
+    };
+
+    reader.readAsText(file);
+  };
+
+  const onUploadChange = (info) => {
+    const { file, fileList } = info;
+    if (fileList.length) {
+      const uploadFile = fileList[0];
+      readFileContent(file.originFileObj, (fileContent: string) => {
+        onChange && onChange(fileContent);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)